{
  "contest": 94,
  "handle": "cccz",
  "address": "0x74d754378a59Ab45d3E6CaC83f0b87E8E8719270",
  "risk": "3",
  "title": "SendValueWithFallbackWithdraw: withdrawFor function may fail to withdraw ether recorded in pendingWithdrawals",
  "issueId": 12,
  "issueUrl": "https://github.com/code-423n4/2022-02-foundation-findings/issues/12"
}